commit 8fbfd3e6d1eccb46cfddcabe4ea0c4860badc424
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Tue Oct 5 09:43:56 2010 -0700

    Staging: brcm80211: remove redefinition of size_t
    
    No driver should ever do this.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/typedefs.h
+++ b/drivers/staging/brcm80211/include/typedefs.h
@@ -28,10 +28,6 @@
 typedef unsigned long long int uintptr;
 #endif
 
-#if defined(_NEED_SIZE_T_)
-typedef long unsigned int size_t;
-#endif
-
 #define TYPEDEF_UINT
 #define TYPEDEF_USHORT
 #define TYPEDEF_ULONG
